code/network/multi_sexp.cpp

```diff
@@ -819,6 +819,6 @@ bool multi_get_ushort(ushort &value)
 void multi_discard_remaining_callback_data()
 {
 	if (!multi_sexp_discard_operator()) {
-		Warning(LOCATION, "Attempt to discard remaining data failed! Callback for operator %d lacks proper termination. Entire packet may be corrupt. Discarding remaining packet"); 
+		Warning(LOCATION, "Attempt to discard remaining data failed! Callback for operator lacks proper termination. Entire packet may be corrupt. Discarding remaining packet");
```

Comment:
It looks like `multi_discard_remaining_callback_data()` is called only by `multi_sexp_set_jumpnode_color()` and `multi_sexp_set_jumpnode_model()`. I'm not entirely clear on what the exact circumstances that can cause the jumpnode name-lookup to fail, though, so I'm not sure what purpose the `multi_discard_remaining_callback_data()` function actually serves; still, if it's only used for a pair of jumpnode-related SEXPs, the warning could probably be made more descriptive of what the actual problem is, assuming anyone actually understands what problem triggers this warning in the first place...